argo-e2e/Dockerfile.buster

```
@@ -0,0 +1,23 @@
+FROM python:slim-buster
+
+RUN set -ex \
+    && apt update \
+    && apt-get install -y --no-install-recommends curl jq unzip \
+    # install aws cli
+    && curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o "awscliv2.zip" \
+    && unzip awscliv2.zip \
+    && ./aws/install \
+    # install argo
+    && curl -sLO https://github.com/argoproj/argo-workflows/releases/download/v3.1.1/argo-linux-amd64.gz \
+    && gunzip argo-linux-amd64.gz \
+    && chmod +x argo-linux-amd64 \
+    && mv ./argo-linux-amd64 /usr/local/bin/argo \
+    # install ngrok
+    && curl -sLO https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip \
+    && unzip ngrok-stable-linux-amd64.zip \
+    && mv ./ngrok /usr/local/bin/ngrok \
```

I think zip will stay after unziping, only gzip will be replaced?
```suggestion
    && mv ./ngrok /usr/local/bin/ngrok \
    && rm ngrok-stable-linux-amd64.zip
```